modules/miscutil/demo/demobibdata.xml

```diff
@@ -26288,4 +26288,56 @@ Only the mountain and I.
     </datafield>
   </record>
 
+  <!----------------------------
+    ARTICLES records used to test 786__w tag
+  ----------------------------->
+
+  <record>
+    <datafield tag="245" ind1=" " ind2=" ">
+      <subfield code="a">Additional data from: Probing color coherence effects in pp collisions at $\sqrt{s}=7\,\text {TeV}$</subfield>
+    </datafield>
+    <datafield tag="786" ind1=" " ind2=" ">
+      <subfield code="w">10</subfield>
+    </datafield>
+    <datafield tag="980" ind1=" " ind2=" ">
+      <subfield code="a">ARTICLE</subfield>
+    </datafield>
+  </record>
```

Comment:
Is it OK that this data record is classified as an ARTICLE?  Shouldn't this be a data collection on its own?